### PR TITLE
Add support for Ruby 2.7.1

### PR DESCRIPTION
**Pull request by Pontus4**

### CODE REVIEW

**Comment by Chrizpy** [comment]
Add support for Ruby platforms with version 2.7.1, see: fa54b07
Mistakes were made though, because previous commit made it so only 2.7.1 platforms were allowed...
This was fixed in: ec30e68

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ rvm:
   - 2.4.9
```
Comment (walro) [inline .travis.yml:10]: Bit off-topic, but mentioning it anyway:

I think we can drop `2.4` from the matrix, as it EoL'd in 2020-03-31. Saves us the time on waiting for it on Travis CI. We could do that in a follow-up PR (where we also bump to the latest patch versions for 2.5 and 2.6)

**Review by walro — APPROVED** [review]
Looks great, merge at will!

File: .travis.yml (hunk 1 above)
Comment (walro) [reply .travis.yml:10]: We did this in 77ddd7487f7757df418b8ff9aad9e6ef7d359727

**Pull request merged**